MSBuildTasks.Project.nuspec

```diff
@@ -9,15 +9,12 @@
     <projectUrl>https://github.com/loresoft/msbuildtasks/</projectUrl>
     <licenseUrl>http://opensource.org/licenses/bsd-license.php</licenseUrl>
     <tags>MSBuild</tags>
+    <developmentDependency>true</developmentDependency>
```

Comment:
This makes it so a project that uses `MSBuild.Community.Tasks` and builds a NuGet package doesn't have a dependency on `MSBuild.Community.Tasks`.